components/molecules/client/user/component.tsx

```diff
@@ -0,0 +1,68 @@
+// components/molecules/client/user/component.tsx
+import React, { useState } from 'react';
+import FollowBtn from '@/components/molecules/client/follow-btn';
+
+interface UserProps {
+  userId: string;
+  postId: string;
+  initialFollowed: boolean;
+  name: string;
+  followersCount: number;
+  bio: string;
+  avatarUrl: string;
+}
+
+const UserComponent: React.FC<UserProps> = ({
+  userId,
+  postId,
+  initialFollowed,
+  name,
+  followersCount,
+  bio,
+  avatarUrl,
+}) => {
+  const [isFollowing, setIsFollowing] = useState(initialFollowed);
+
+  const handleFollowToggle = async () => {
+    const action = isFollowing ? 'remove' : 'add';
+    try {
+      const response = await fetch(`/users/${userId}/reading-list`, {
```

Comment:
How come this component calling the reading list endpoint? I don't think that exists yet. For now, just handle state, don't make any fetch requests since the backend doesn't have this endpoint yet (afaik).

Reply:
I think this is how to handle state, not sure